subsys/nrf_rpc/include/nrf_rpc_utils.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef UTILS_NRF_RPC_UTILS_H_
+#define UTILS_NRF_RPC_UTILS_H_
+
+const char *get_version();
```

Comment:
So we're not putting this in a sample after all? Why? If we're willing to make it an official NCS library, and not part of the samples, this requires:
- a proper documentation (both doxygen and RST) as I have no idea which version this is supposed to return.
- a better name as "nRF RPC utils" can mean anything. So that future devs know what this library is meant for. Something like "nRF RPC device information" might work.